### PR TITLE
Enable expandable z-index customization by using Custom Property (aka CSS Variable)

### PR DESCRIPTION
**Pull request by tekreme73**
For #29

### CODE REVIEW

**Comment by alshakero** [comment]
That's a great idea! Thanks a lot! Can you please document that in the README?

File: src/juicy-popover.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
             position: absolute;
             top: 0;
             left: 0;
-            z-index: 999;
+            z-index: 999; /*For IE11 support*/
```
Comment (alshakero) [inline src/juicy-popover.html]: Please remove this because we use ES6 all over the element. It doesn't support IE in the first place.

**Comment by tekreme73** [comment]
I removed the IE11 fallback and document this custom property to the README.md file. I don't know if there is other things to do according to this update (make automated tests? demo?)

**Comment by alshakero** [comment]
No that's more than enough! Thanks a million indeed.

**Pull request merged**